src/node_report_module.cc

```diff
@@ -119,13 +119,13 @@ static void SetSignal(const FunctionCallbackInfo<Value>& info) {
 static void ShouldReportOnFatalError(const FunctionCallbackInfo<Value>& info) {
   Environment* env = Environment::GetCurrent(info);
   info.GetReturnValue().Set(
-      env->isolate_data()->options()->report_on_fatalerror);
+    node::per_process::cli_options->report_on_fatalerror);
```

Comment:
Fwiw, the previous indentation of 4 spaces here was correct

Reply:
@addaleax, Fixed it.